rules/eslint/node/walmart.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
   "rules": {
     // enforce return after a callback
     "callback-return": 2,
+    // disallow require() outside of the top-level module scope
+    "global-require": 0,
```

Comment:
It's conventional and often enforced in other languages. I don't see any reason to buck the trend in JS land just because we can.